internal/analytics/segment.go

```diff
@@ -0,0 +1,83 @@
+package analytics
+
+import (
+	"github.com/porter-dev/porter/internal/logger"
+	segment "gopkg.in/segmentio/analytics-go.v3"
+)
+
+type AnalyticsSegmentClient interface {
+	Identify(SegmentIdentifier) error
+	Track(SegmentTrack) error
+}
+
+type AnalyticsSegment struct {
+	segment.Client
+	isEnabled bool
+	logger    *logger.Logger
+}
+
+//
+/*
+	Initialize the segment client and return a superset of it, the AnalyticsSegmentClient will handle cases when
```

Comment:
```suggestion
// Initialize the segment client and return a superset of it, the AnalyticsSegmentClient will handle cases when
// the segment client failed on initialization or not enabled
```

Can you modify these comments to keep the style consistent with the other parts of the code?